### DISCUSSION TITLE
User profile repo buttons break on resolutions <1044px

### DISCUSSION THREAD

**Discussion by b4u7** [community/community#38773] [Projects and Issues]
### Select Topic Area

Bug

### Problem:
When accessing a user's profile, the buttons on the repo list break on most resolutions with a width below 1044px.

![unnamed](https://user-images.githubusercontent.com/10912932/201097097-93a54c8f-bef4-4077-82b4-2b3a90411e85.png)

### Reproduction Steps:
```
1. Access any users page
2. Set the width of the page below 1044px.
```

### Browser:
Firefox 107.0b9 (64-bit)

### Example Url:
https://github.com/b4u7?tab=repositories

**Comment by ossanna16** [comment]
Hi there @b4u7 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by jfullstackdev** [comment]
yes of course, 

![image](https://user-images.githubusercontent.com/47092464/201474569-f0fc3695-a1ba-4861-af07-8625e1f551d6.png)

but to give you a background, GitHub improved a lot from the past of the UI being responsive 

maybe they will be implementing the full mobile-responsive but at the least priority because mostly GitHub is accessed using standard productivity computers

also, even GitHub Mobile has a lot to be improved too
